eng/common/TestResources/deploy-test-resources.yml

```diff
@@ -43,7 +43,7 @@ steps:
 
   - ${{ if eq('true', parameters.UseFederatedAuth) }}:
     - task: AzurePowerShell@5
-      displayName: Deploy test resources
+      displayName: 🚀 Deploy test resources
```

Comment:
This change breaks ASCII validation in C++ pipelines.
For example see https://dev.azure.com/azure-sdk/public/_build/results?buildId=4125343&view=logs&j=00048e40-e66a-5148-a583-db054fe16a39&t=4fca78b1-cd06-5541-9843-79f482823f37&l=17 from https://github.com/Azure/azure-sdk-for-cpp/pull/5970

Can we revert adding the emoji?

cc @LarryOsterman, @benbp

Reply:
FWIW,  @weshaggard fixed the validation in the C++ pipelines about 15 minutes after the issue was discovered.

There is no need to revert this change.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/pull/5971

Reply:
@ahsonkhan was there a specific reason that ascii validation needed to be added to the platform matrix and run in a build job? Otherwise I plan on moving it to analyze.

Reply:
Probably out of simplicity/convenience at the time. It's from a while back, so not sure.
What's the benefit of moving to analyze vs running in a build job?
How are those different? I am not familiar with "analyze".

Reply:
Typically analyze is good for tasks like this that aren't actually related to a specific OS/version combination. That way when failures happen it's more obvious at a glance whether they are related to higher level checks like spelling or actual test failures for a specific platform. And it also keeps us from having too many conditionals in the pipeline yaml which can become hard to follow. In this case it will simplify the matrix config too.